pkg/ddevapp/templates.go

```diff
@@ -62,8 +62,8 @@ const ConfigInstructions = `
 # Alternatively, an explicit Composer version may be specified, for example "2.2.18".
 # To reinstall Composer after the image was built, run "ddev debug refresh".
 
-# nodejs_version: "16"
-# change from the default system Node.js version to another supported version, like 12, 14, 17, 18.
+# nodejs_version: "18"
+# change from the default system Node.js version to another supported version, like 12, 14, 17, 18, 19, 20.
```

Comment:
We'll want to get rid of 12 and 17, right? 

I'm also not sure why we would do 19, since you can easily do it with nvm. If it's "supported" I'm OK with it.

Reply:
```suggestion
# change from the default system Node.js version to another supported version, like 16, 18, 19, 20.
```